### ISSUE TITLE
podspec specifies iOS 10.0 as the minimum deployment target

### ISSUE THREAD

**Issue by ataibarkai** [mauriciosantos/Buckets-Swift#5]
Hello,

The podspec specifies iOS 10.0 as the minimum deployment target, while the readme specifies iOS 9.
Is this in error?

**Comment by ataibarkai** [comment]
Hey @mauriciosantos I noticed that you fixed this issue on master and bumped the version -- thanks for that!

Did you notice that the CI job had failed? I assume it's responsible for creating a new release?

**Comment by mauriciosantos** [comment]
yes. i tried to make a quick fix. I haven't had much time to work on it.

**Comment by ataibarkai** [comment]
Gotcha. Let me know if I can help

**Comment by sghiassy** [comment]
Noticed the same problem. Until its fixed, you can simply reference the `SHA` in your Podfile to pull in the updated version

```ruby
pod 'Buckets', :git => 'https://github.com/mauriciosantos/Buckets-Swift.git', :commit => 'bb022bad2a3bd7921af47492124965687a9330e7'
```
